docs/reference/kubectl/cheatsheet.md

```diff
@@ -215,7 +215,7 @@ $ kubectl logs -f my-pod                              # stream pod logs (stdout)
 $ kubectl logs -f my-pod -c my-container              # stream pod container logs (stdout, multi-container case)
 $ kubectl run -i --tty busybox --image=busybox -- sh  # Run pod as interactive shell
 $ kubectl attach my-pod -i                            # Attach to Running Container
-$ kubectl port-forward my-pod 5000:6000               # Forward port 6000 of Pod to your to 5000 on your local machine
+$ kubectl port-forward my-pod 5000:6000               # Listen on port 5000 on the local machine and forward to port 5000 in my-pod
```

Comment:
You've got a typo here. Should be "Listen on port 5000 on the local machine and forward to port 6000 on my-pod"